src/Tribe/Main.php

```diff
@@ -4897,5 +4883,35 @@ public function maybe_set_currency_position_with_post( $reverse_position, $post_
 
 			return $reverse_position;
 		}
+
+		/**
+		 * Instances all classes that should be built at `plugins_loaded` time.
+         *
+         * Classes are bound using the `tribe_singleton` function before and then
+         * built calling the `tribe` function.
+		 */
+		public function on_plugins_loaded(  ) {
+			tribe_singleton( 'events-aggregator.main', 'Tribe__Events__Aggregator' );
+			tribe_singleton( 'events.shortcode.event-details', 'Tribe__Events__Shortcode__Event_Details', array( 'hook' ) );
+			tribe_singleton( 'events.ignored-events', 'Tribe__Events__Ignored_Events' );
+
+			tribe( 'events-aggregator.main' );
```

Comment:
I've thought that there might be more `aggregator`s in the future (venue, organizers, ticktes, whatever)